modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -2769,36 +2781,36 @@ private void checkIncrementalCanBeCreated(
                     "Encrypted cache groups not supported [groupId=" + grpId + ']');
             }
 
-            File snpCacheDir = sft.cacheStorage(gctx.config());
-
-            if (!snpCacheDir.exists()) {
-                throw new IgniteCheckedException("Create incremental snapshot request has been rejected. " +
-                    "Cache group directory not found [groupId=" + grpId + ']');
-            }
+            for (File snpCacheDir : sft.cacheStorages(gctx.config())) {
+                if (!snpCacheDir.exists()) {
+                    throw new IgniteCheckedException("Create incremental snapshot request has been rejected. " +
+                        "Cache group directory not found [groupId=" + grpId + ']');
```

Comment:
Please add a dir name to exception message, otherwise we won't know what directory hasn't been found exactly.

Reply:
Done